### PR TITLE
Pull arch specific client and install images

### PR DESCRIPTION
**Pull request by manojnkumar**
Fixes: #2660

### CODE REVIEW

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -518,9 +518,17 @@ def expose_ocp_version(version):
         str: Version of OCP exposed to full version if latest nighly passed
 
     """
+    ocp_arch = config.DEPLOYMENT['ocp_arch']
     if version.endswith(".nightly"):
+        if ocp_arch == "s390x":
+            path = "https://openshift-release-s390x.apps.ci.l2s4.p1.openshiftapps.com/"
```
Comment (ebenahar) [inline ocs_ci/utility/utils.py:524]: this URL should not be saved here..
@petr-balogh should it be loaded from a conf file?

Same for the other URLs

Comment (manojnkumar) [reply ocs_ci/utility/utils.py:524]: @ebenahar @petr-balogh : Made this a configuration option.

**Comment by manojnkumar** [comment]
@ebenahar @petr-balogh : Is this what you were looking for? To pick up paths from the config file, and the default be for x86_64?

**Comment by lsmcfadden** [comment]
@ebenahar @petr-balogh @manojnkumar ?

**Comment by lsmcfadden** [comment]
@RazTamir can anyone on your team review/comment?

**Comment by vasukulkarni** [comment]
This is fixed, there was another issue from s390 team, I will try to get that PR reference.

**Comment by vasukulkarni** [comment]
we discussed this briefly in our meeting,  we dont have to modify existing config items defined in config file. Refer this PR for details on how to get s390 working https://github.com/red-hat-storage/ocs-ci/pull/3882/files . The issue was from: https://github.com/red-hat-storage/ocs-ci/issues/3772

**Comment by petr-balogh** [comment]
@vasukulkarni I looked at this briefly and the solution provided in the doc is for specific versions we found out in https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/ which is mainly GAed versions or some beta versions available there as well but it doesn't cover nightlies. Not sure if there are available nightlies also for s390 arch. If so then I prefer to have configuration file with value for nightlies for S390 arch or have some logic to check the arch and dynamically change the value to that one if running on s390.    @manojnkumar @lsmcfadden  are you aware from which place to get the nightly OCP builds for your arch?

**Comment by petr-balogh** [comment]
@manojnkumar please this one needs rebase and finish some discussion here.  Are you going to finish with this PR? If not, please close.

**Comment by manojnkumar** [comment]
@petr-balogh @vasukulkarni  The url_template documentation seems sufficient for the purposes of other arches as in this PR: https://github.com/red-hat-storage/ocs-ci/pull/3882/files

I do not believe we have nightly builds of the `oc` client and `openshift-install` binaries available externally for other arches (afaik).

**Comment by petr-balogh** [comment]
Thanks, based on comment above I am closing this one PR.

**Pull request closed without merging**